src/pages/AppProfile.vue

```diff
@@ -225,6 +228,8 @@ export default {
     const organisationDetails = ref({
       name: " ",
       size: 0,
+      // Nothing else can really be invalid, but this needs to be a systematic thing
+      sizeErr: null,
```

Comment:
Prefer fully qualified names over shorthands.

```suggestion
      sizeErrorMessage: null,
```

Reply:
Agreed.